utils/keycodes.js

```diff
@@ -90,3 +90,19 @@ export const displayShortcut = mapValues( modifiers, ( modifier ) => {
 		return shortcut.replace( /⌘\+([A-Z0-9])$/g, '⌘$1' );
 	};
 } );
+
+export const is = mapValues( modifiers, ( getModifiers ) => {
```

Comment:
This is a bit of a confusing function name to me. "is" what? I get it works like `is.primary( 'a' )` but that isn't explained in a comment/docblock here and that's a bit of an unexpected API to me. Maybe `matchesKeycode` or something? Or a doc block here to make this a bit easier to understand without gripping through the codebase for usage (which is what I needed to do).

Reply:
It checks more than just the `keyCode`, so `matchesKeycode` feels a bit weird. What we want to do is check if the keydown/keyup event matches the modifiers and letter. Let me think. :)

Reply:
Maybe something like `matchesShortcut`? Or `KeyboardEventIsPrimary`?

Reply:
How about `isKeyboardEvent.primary( 'a' )`?